src/parser/shaman/elemental/modules/talents/Aftershock.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { Options } from 'parser/core/Analyzer';
+
+import { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import Events, { EnergizeEvent } from 'parser/core/Events';
+import Statistic from 'interface/statistics/Statistic';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ResourceGenerated from 'interface/others/ResourceGenerated';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+
+class Aftershock extends Analyzer {
+  refund: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.AFTERSHOCK_TALENT.id);
+
+    this.addEventListener(Events.energize.spell(SPELLS.AFTERSHOCK), this.onAftershock);
+  }
+
+  onAftershock(event: EnergizeEvent) {
+    this.refund += event.resourceChange;
+  }
+
```

Comment:
You don't want to track anything wasted here?

Reply:
By the nature of it just being a refund there should never be any waste.